packages/e2e/test/artillery/wallet-restoration/WalletRestoration.ts

```diff
@@ -2,21 +2,17 @@ import { AddressType, GroupedAddress, util } from '@cardano-sdk/key-management';
 import { AddressesModel, WalletVars } from './types';
 import { Cardano } from '@cardano-sdk/core';
 import { FunctionHook } from '../artillery';
-import { Pool, QueryResult } from 'pg';
 import { StubKeyAgent, getEnv, getWallet, walletVariables } from '../../../src';
-import { findAddressesWithRegisteredStakeKey } from './queries';
 import { logger } from '@cardano-sdk/util-dev';
-import { walletReady } from '../../util';
-
-const env = getEnv([
-  ...walletVariables,
-  'DB_SYNC_CONNECTION_STRING',
-  'ARRIVAL_PHASE_DURATION_IN_SECS',
-  'VIRTUAL_USERS_COUNT'
-]);
+import { waitForWalletStateSettle } from '../../util';
 
+const env = getEnv([...walletVariables, 'ARRIVAL_PHASE_DURATION_IN_SECS', 'VIRTUAL_USERS_COUNT']);
 const operationName = 'wallet-restoration';
-const SHORTAGE_OF_WALLETS_FOUND_ERROR_MESSAGE = 'Addresses found from db are less than desired wallets count';
+
+const networkNameMap = {
+  [Cardano.NetworkId.Testnet]: 'testnet',
```

Comment:
```suggestion
  [Cardano.NetworkId.Preprod]: 'preprod',
```
... and also the _preprod_ file can be added

Reply:
Just saw you extracted data for `preprod` as well. Going to replace `testnet` with `preprod` :)

Reply:
28ceb039981803fb27129eafea78f41868f766e3